frontend/src/less/main.less

```diff
@@ -885,47 +889,6 @@ div:focus {
   }
 }
 
-/* Upgrade */
-
-.planPanel {
-  padding: 10px 20px;
-  border-radius: 5px;
-  border-width: 1px;
-  border-style: solid;
-  vertical-align: top;
-  width: 32%;
-  .transition(all 0.2s ease);
-  cursor: pointer;
-  &.muted,
-  &.muted:hover {
-    cursor: default;
-    box-shadow: none;
-  }
-}
-
-.offerTag {
-  color: @linkgreen;
-  font-size: 12px;
-  svg {
-    width: 0.7em;
-    height: 0.7em;
-    vertical-align: middle;
-    margin-left: 4px;
-    margin-right: 2px;
-  }
-}
-
-.offerBox {
-  background-color: #337a87;
-  border-radius: 4px;
-  padding: 8px 12px;
-  color: @defaultBackground;
-  * {
-    color: inherit;
-    font-size: inherit;
-  }
-}
```

Comment:
This is still in use in the `OfferHeader` component - even though I'm not sure when we'll have another offer